MapboxNavigation/NavigationMapView.swift

```diff
@@ -404,13 +397,12 @@ open class NavigationMapView: MGLMapView {
         
         let minimumZoomLevel: Float = 14.5
         
-        let shaftLength = max(min(50 * metersPerPoint(atLatitude: maneuverCoordinate.latitude), 50), 10)
+        let shaftLength = max(min(30 * metersPerPoint(atLatitude: maneuverCoordinate.latitude), 30), 10)
         let polyline = Polyline(routeCoordinates)
-        let shaftCoordinates = Array(polyline.trimmed(from: maneuverCoordinate, distance: -shaftLength / 2).coordinates.reversed()
```

Comment:
@bsudekum 

> isn't this already happening?

I guess using the entire shaft length here instead of half of it fixes #648?